From: Fedora/Cygwin
Subject: Enable SYS_BASH_LOGOUT

Enables sourcing of /etc/bash.bash_logout for login shells on exit.
Allows system administrators to perform cleanup tasks.

--- a/config-top.h
+++ b/config-top.h
@@ -94,7 +94,7 @@ typedef void *sh_parser_state_t;
 /* #define SYS_BASHRC "/etc/bash.bashrc" */
 
 /* System-wide .bash_logout for login shells. */
-/* #define SYS_BASH_LOGOUT "/etc/bash.bash_logout" */
+#define SYS_BASH_LOGOUT "/etc/bash.bash_logout"
 
 /* Define this to make non-interactive shells begun with argv[0][0] == '-'
    run the startup files when not in posix mode. */
